### DISCUSSION TITLE
MarkdownSection from Incubation Lab compile question

### DISCUSSION THREAD

**Discussion by tranqy** [microsoft/fluentui-blazor#1019] [Q&A]
I'm trying to get the MarkdownSection working. 

[MarkdownSection](https://www.fluentui-blazor.net/Lab/MarkdownSection)

I've created a new Blazor project and gotten FluentUI components to work. 

I tried to put the c# and Blazor code into a .blazor component but get the following error.

> Severity	Code	Description	Project	File	Line	Suppression State
> Error (active)	CS0246	The type or namespace name 'IStaticAssetService' could not be found (are you missing a using directive or > an assembly reference?)	Components\Partials\MarkdownSection.razor	14	

I've tried to find where that type is.  When I bring in the old FAST version I can compile but get a runtime error. 

Any thoughts?

**Comment by vnbaaij** [comment]
Hi,

We actually moved the `IStaticAssetService` to the demo project. It lives in the `FluentUI.Demo.Shared` namespacenow (in the `Infrastructure` folder). 

You can remove that fuctionality from your code as it is quite specific to our needs. To remove, delete line 16 and 17, remove the `FromAsset` parameter and remove line 67-70 in `OnAfterRenderAsync` (all in MarkdownSection.razor.cs`)

If you want something like https://www.fluentui-blazor.net/Lab/MarkdownSection#markdownfromanasset to work, you need to copy almost all the code from the Infrastructure folder to your project.
